meshpy/abaqus/beam.py

```diff
@@ -31,10 +31,8 @@
 """This file provides functions to create Abaqus beam element classes to be
 used with MeshPy."""
 
-# Python modules
```

Comment:
I also really like that these comments (that I introduced back in the day...) are gone!